dns/zone-configs/go.canary.k8s.io.yaml

```diff
@@ -0,0 +1,57 @@
+---
+# The go.k8s.io zone
+# URL shortener service
+'':
+  type: CNAME
+  value: txtd.io.
+
+# TXTDirect data for shortener
+# Docs: https://about.txtdirect.org/docs/specification/#host-type
+_redirect:
```

Comment:
I don't understand why some are `_redirect.foo` and some are `redirect.foo` ?

Can you please include comments to explain to people who are reading/editing this without being experts.

Reply:
Should always be `_redirect`. Fixing on next push.